css/_uswds-theme-custom-styles.scss

```diff
@@ -1478,7 +1482,7 @@ code[class*="language-plaintext"] {
 
 .usa-button,
 .link-download,
-.media_link {
+.media__link {
```

Comment:
The class changed, but the markup still references the old. Please update in markup.